src/ngraph/ngraph_utils.h

```diff
@@ -29,6 +28,7 @@
 #include <vector>
 
 #include "ngraph_graph.h"
+#include "mxnet/ndarray.h"
```

Comment:
I think this reorder will break `make cpplint`

Reply:
Actually it does.  I try to always run `make cpplint` prior to creating a PR.  

Should we add `make cpplint` this to our CI pre-merge tests?

Reply:
Yeah, we should.